.github/workflows/release.yaml

```diff
@@ -10,7 +10,7 @@ env:
 
 jobs:
   create-release:
-    runs-on: ubuntu-20.04
```

Comment:
This is related to minimal supported libc version so this version should be low as possible. It should be `ubuntu-22.04`. Unless we drop the ubuntu-22 support.

Reply:
✔️